src/modules/requirements/TimeRequirement.ts

```diff
@@ -1,33 +1,26 @@
 import { AchievementOption } from '../GameConstants';
 import GameHelper from '../GameHelper';
 import Requirement from './Requirement';
+import DayCyclePart from '../dayCycle/DayCyclePart';
+import DayCycle from '../dayCycle/DayCycle';
 
 export default class TimeRequirement extends Requirement {
     private updateTrigger = ko.observable(0);
-    constructor(public startHour: number, public endHour: number, option = AchievementOption.more) {
+    constructor(public dayCycleParts: DayCyclePart[], option = AchievementOption.more) {
         super(1, option);
         setInterval(
             () => GameHelper.incrementObservable(this.updateTrigger),
             60 * 1000,
         );
```

Comment:
There shouldn't be any need for this setInterval or the updateTrigger property if the `DayCycle.currentCyclePart` observable is updating properly